client/landing/gutenboarding/path.ts

```diff
@@ -113,23 +115,70 @@ export function useIsAnchorFm(): boolean {
 // Returns the anchor podcast id. First looks in "location state",
 // provided by react-router-dom, if not available there, checks the query string.
 export function useAnchorFmPodcastId(): string | null {
-	const { state: locationState, search } = useLocation< GutenLocationStateType >();
-	const sanitizePodcastId = ( id: string ) => id.replace( /[^a-zA-Z0-9]/g, '' );
+	const sanitize = ( id: string ) => id.replace( /[^a-zA-Z0-9]/g, '' );
+	return useAnchorParameter( {
+		queryParamName: 'anchor_podcast',
+		locationStateParamName: 'anchorFmPodcastId',
+		sanitize,
+	} );
+}
+
+// Returns the anchor podcast id. First looks in "location state",
```

Comment:
Tiny edit, should this say episode id?

Reply:
Fixed here [add/anchor-gutenboarding-episode-id 3e3c3066ec] Fix comment